htdocs/landing/aup.html

```diff
@@ -0,0 +1,104 @@
+<!doctype html>
+<html>
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  <title>GOCDB Privacy Notice</title>
+  <link rel="stylesheet" type="text/css" href="home.css" />
+  <link rel="SHORTCUT ICON" href="Logo-1.4-FavIcon-32x32.regional.ico" />
+</head>
+
```

Comment:
I feel it would be useful to make it clear near the top of the page that this is not the definitive source version of the AUP and that the actual version can be found wherever.

However, if the version here is the definitive version for GOCDB, then we should make it clear how this document relates to the one hosted by EGI.

What happens when this version differs from the EGI hosted one?

Similar comments apply to the Priv. Not.

Reply:
Agreed, it is not clear. I think this is the definitive version (but should be kept in step with EGI). Workflow should be (but is not) in place to inform the user if it changes. The same scenario plays for VOMS afaiks. Propose to change to along the lines of "Your use of the Grid Operations Centre Database is governed by the EGI Acceptable Use Policy and Conditions of Use (version blah) reproduced below and available from here <link>.". We might add "..you will be notified of changes.." etc. but, until support workflow is in place I'd say not. Reasonable?